pkg/cli/environment.go

```diff
@@ -189,7 +189,7 @@ func envCSV(name string) (ls []string) {
 
 func (s *EnvSettings) EnvVars() map[string]string {
 	envvars := map[string]string{
-		"HELM_BIN":               os.Args[0],
+		"HELM_BIN":               envOr("HELM_BIN", os.Args[0]),
```

Comment:
```suggestion
		"HELM_BIN":               os.Args[0],
```

Reply:
> Would it be all right if I revert the line and remove those test cases?

I think the test cases should be kept to demostrate the functionality works (and ensure it continues to work into the future)

> Unfortunately, I have been able to find the way to solve this FAILs.

It seems like the code could be structured differently? To e.g. pass the value of the helm binary path into the logic. Or, the output could be templated/formatted to avoid requiring a static string?